test/DynamoCoreTests/SerializationTests.cs

```diff
@@ -666,6 +667,33 @@ public void SerializationTest(string filePath)
                 serializationTestUtils.SaveWorkspaceComparisonData);
         }
 
+        /// <summary>
+        /// This parameterized test finds all .dyn files in directories within
+        /// the test directory, opens them and executes, then converts them to
+        /// json and executes again, comparing the values from the two runs
+        /// while being in a different culture.
+        /// </summary>
+        /// <param name="filePath">The path to a .dyn file. This parameter is supplied
+        /// by the test framework.</param>
+        [Test, TestCaseSource("FindWorkspaces"), Category("JsonTestExclude")]
+        public void SerializationTestInDifferentCulture(string filePath)
```

Comment:
@alfarok @smangarole is there any consequence for cogitator tests to have another serializationTest which is potentially saving more json graphs?

@sm6srw maybe you could isolate these tests by creating some other folder that the files are saved to - I know that the test files and other associated data from the serialization tests are uploaded to cogitator- and I just want to make sure this does not have unintended consequences.

I believe @alfarok and @smangarole wrote the s3 upload script and will know more.

Reply:
I believe @gregmarr was manually copying the contents from s3 so it wouldn't immediately have any influence but may in the future.  Greg would probably have the best perspective, not sure if he made any changes since.

Reply:
Thanks, I wasn't aware of the directory. I will use a new directory for this test.

Reply:
We have scripts for pulling, comparing, and updating, so it's automated, but not automatic in that we run them manually.